src/garage/envs/multi_env_wrapper.py

```diff
@@ -124,6 +145,11 @@ def active_task_index(self):
             int: Index of active task.
 
         """
+        if hasattr(self.env, 'active_task_index'):
+            return self.env.active_task_index
+        else:
+            return self._active_task_index
+
         return self._active_task_index
```

Comment:
The field self._active_task_index will be potentially different when using meta world environments. The private field acts as a way to index into env_names and envs

Reply:
By saying _**using metaworld environments**_, are you referring to use `MT10.get_train_tasks()` and `MT10.from_task(list_of_tasks)`? In such cases, MultiEnvWrapper saves only one instance in its `self._task_envs`, so that `self._active_task_index` will always be 0.